frontend/src/ui-components/AcmSearchInput/AcmSearchInput.tsx

```diff
@@ -0,0 +1,273 @@
+                        value={constraint.columnId}
+                        placeholder={t('Select a column')}
+                        onChange={(columnId) => {
+                          const newConstraintArray = [...searchConstraints]
+                          newConstraintArray[index].columnId = columnId
+                          setSearchConstraints(newConstraintArray)
+                          setActiveConstraints && setActiveConstraints(newConstraintArray)
+                        }}
+                      >
+                        {getAvailableColumnNames()}
+                      </AcmSelect>
+                    </FlexItem>
+                    <FlexItem style={{ width: '35%' }}>
+                      <AcmTextInput
+                        label={t('Value')}
+                        placeholder={t('Enter a value')}
+                        id="search-value"
+                        value={constraint.value}
+                        onChange={(newValue) => {
+                          const newConstraintArray = [...searchConstraints]
+                          newConstraintArray[index].value = newValue
+                          setSearchConstraints(newConstraintArray)
+                          setActiveConstraints && setActiveConstraints(newConstraintArray)
+                        }}
+                      />
+                    </FlexItem>
+                    <FlexItem>
+                      <Button
+                        id="remove-constraint"
+                        style={{ marginTop: '2em' }}
+                        variant={ButtonVariant.link}
+                        onClick={(event) => {
+                          const newConstraintArray = [...searchConstraints]
+                          newConstraintArray.splice(index, 1)
+                          setSearchConstraints(newConstraintArray)
+                          setActiveConstraints && setActiveConstraints(newConstraintArray)
+                          event?.stopPropagation()
+                        }}
+                        aria-label={t('Remove constraint')}
+                        icon={<TimesCircleIcon id="remove-constraint-icon" />}
+                      />
+                    </FlexItem>
+                  </Flex>
+                )
+              })}
+              {canAddConstraints && (
+                <Button
+                  isDisabled={disableAddConstraint || searchConstraints?.length > 1}
+                  variant={ButtonVariant.link}
+                  onClick={(event) => {
+                    event.stopPropagation()
+                    searchConstraints &&
+                      setSearchConstraints([
+                        ...searchConstraints,
+                        { operator: SearchOperator.Equals, value: '', columnId: '' },
+                      ])
+                  }}
+                  icon={<PlusCircleIcon />}
+                >
+                  {t('Add a search constraint').toString()}
```

Comment:
```suggestion
                  {t('Add a search constraint')}
```
Good idea to always test with `npm run plugins`. It finds more problems in general and more problems that occur when running as a plugin.

You imported `t` from `i18next`; need to use the `useTranslation` hook.

<img width="1511" alt="image" src="https://github.com/user-attachments/assets/52508f14-b0aa-44fd-bbd8-d1d8f9fa8ab9">